integrated_channels/blackboard/exporters/learner_data.py

```diff
@@ -79,11 +83,15 @@ def get_learner_assessment_data_records(
         """
         if enterprise_enrollment.enterprise_customer_user.user_email is None:
             # We need an email to find the user on blackboard.
-            LOGGER.debug(
-                'No learner data was sent for LMS User [%s] because Blackboard User ID not found for customer [%s]',
+            LOGGER.debug(generate_formatted_log(
+                'blackboard',
+                enterprise_enrollment.enterprise_customer_user.enterprise_customer.uuid,
                 enterprise_enrollment.enterprise_customer_user.user_id,
-                enterprise_enrollment.enterprise_customer_user.enterprise_customer.name
-            )
+                None,
+                ('get_learner_assessment_data_records finished. No learner data was sent because '
+                'Blackboard User ID not found for [{name}]'.format(
+                    name=enterprise_enrollment.enterprise_customer_user.enterprise_customer.name
```

Comment:
Isn't this if block referring to the individual user?
`enterprise_enrollment.enterprise_customer_user.user_email`
Thus shouldn't this be like `Blackboard User ID not found for [lms user id, so we aren't using a user's real name?]` It definitely cancels out of the learner assessment audit altogether, but I _think_ it's just the block for a specific user still?

Reply:
I know the generated formatted log standardizes including the user id, but just reading this log is confusing, since it implies the entire enterprise's learner data fails because _the enterprise_ doesn't have a user id in blackboard.

Reply:
So I think bulk assessment level jobs would also hit this same log line and I'm not entirely sure it would cancel out of the entire job or just return nothing for this single user. So I'm still leaning into rewording this line so it doesn't insinuate the entire job fails, ie maybe: `No learner data was sent *for this user* because a Blackboard User ID could not be found for them.` Since we prefix user id and Customer Name, we probably don't need the log line itself to contain those contents as well. I'm also realizing if we want an easy way to determine if certain errors are happening at high rates, we can actually track those numbers better with a standardized log message. (ie we can use the full error message as a search term, instead of being able to only pull part of it or have to use regex to parse out the dynamic user ids, names, etc.)